core-services/image-mirroring/knative/mapping_knative_v0_21_quay

```diff
@@ -11,13 +11,19 @@ registry.ci.openshift.org/openshift/knative-v0.21.0:knative-eventing-test-record
 registry.ci.openshift.org/openshift/knative-v0.21.0:knative-eventing-test-request-sender quay.io/openshift-knative/request-sender:v0.21.0
 registry.ci.openshift.org/openshift/knative-v0.21.0:knative-eventing-test-request-sender quay.io/openshift-knative/request-sender:v0.21
 registry.ci.openshift.org/openshift/knative-v0.21.0:knative-eventing-test-wathola-fetcher quay.io/openshift-knative/wathola-fetcher:v0.21.0
-registry.ci.openshift.org/openshift/knative-v0.21.0:knative-eventing-test-wathola-fetcher quay.io/openshift-knative/wathola-fetcher:v0.21
 registry.ci.openshift.org/openshift/knative-v0.21.0:knative-eventing-test-wathola-forwarder quay.io/openshift-knative/wathola-forwarder:v0.21.0
-registry.ci.openshift.org/openshift/knative-v0.21.0:knative-eventing-test-wathola-forwarder quay.io/openshift-knative/wathola-forwarder:v0.21
 registry.ci.openshift.org/openshift/knative-v0.21.0:knative-eventing-test-wathola-receiver quay.io/openshift-knative/wathola-receiver:v0.21.0
-registry.ci.openshift.org/openshift/knative-v0.21.0:knative-eventing-test-wathola-receiver quay.io/openshift-knative/wathola-receiver:v0.21
 registry.ci.openshift.org/openshift/knative-v0.21.0:knative-eventing-test-wathola-sender quay.io/openshift-knative/wathola-sender:v0.21.0
-registry.ci.openshift.org/openshift/knative-v0.21.0:knative-eventing-test-wathola-sender quay.io/openshift-knative/wathola-sender:v0.21
+registry.ci.openshift.org/openshift/knative-v0.21.1:knative-eventing-kafka-test-wathola-fetcher quay.io/openshift-knative/wathola-fetcher:v0.21.1
+registry.ci.openshift.org/openshift/knative-v0.21.1:knative-eventing-kafka-test-wathola-fetcher quay.io/openshift-knative/wathola-fetcher:v0.21
+registry.ci.openshift.org/openshift/knative-v0.21.1:knative-eventing-kafka-test-wathola-forwarder quay.io/openshift-knative/wathola-forwarder:v0.21.1
```

Comment:
`quay.io/openshift-knative/eventing-kafka-wathola-forwarder` is a better pre fix